.github/workflows/publish.yaml

```diff
@@ -0,0 +1,34 @@
+name: Lint, test, build, and publish
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publish:
+    name: Publish regular version
+    if: "!contains(github.ref, 'beta')"
+    runs-on: ubuntu-latest
+    env:
+      CLOUDSMITH_TOKEN: ${{ secrets.CLOUDSMITH_TOKEN }}
```

Comment:
I think we publish our non-private packages to NPM instead of CloudSmith.

Reply:
We could change that however? Not sure what the other @Ultimaker/stardust_frontend peeps think.

Reply:
Ah right, for now I'll add a distinction between the NPM and CloudSmith packages.

Reply:
I do not know enough about the public packages that Ultimaker has to judge this to be honest.